ui.tests/test-module/libs/commons/guideSelectors.js

```diff
@@ -170,7 +170,10 @@ var selectors = {
             editRule : "#EditableToolbar [data-action='editexpression']",
             createRuleButton : "#create-rule-button",
             saveRule : ".exp-Save-Button",
-            closeRuleEditor : ".exp-Close-Button"
+            closeRuleEditor : ".exp-Close-Button",
+            sideToggleButton : "#exp-toggle-sidepanel-button",
+            formObjectsTab : ".exp-sidepanel coral-tabview coral-tab coral-tab-label:first",
```

Comment:
Is it possible to have a specific selector without having coral dependency ?

Reply:
There is no specific selector available for the tabs currently. So we will have to depend upon the coral tag.